lib/easypost/services/event.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'json'
+
+# Webhook Events are triggered by changes in objects you've created via the API.
+class EasyPost::Services::Event < EasyPost::Services::Service
+  MODEL_CLASS = EasyPost::Models::Event
+
+  # Retrieve an Event object
+  def retrieve(id)
+    @client.make_request(:get, "events/#{id}", MODEL_CLASS)
+  end
+
+  # Retrieve all Event objects
+  def all(params = {})
+    @client.make_request(:get, 'events', MODEL_CLASS, params)
+  end
+
+  # Converts a raw webhook event into an EasyPost object.
+  def receive(values)
```

Comment:
issue: this needs to be in the utils and not called against the client since it's just a local function. It also needs to be titled `receive_event`